runtime/server/x86/frontend/BUILD

```diff
@@ -19,8 +19,8 @@ cc_library(
     srcs = ['feature_pipeline.cc'],
     hdrs = ['fbank.h', 'feature_pipeline.h'],
```

Comment:
Please refer to what `machine_learning/one` and `speech/hotword` are doing on gerrit.

For consistency, a list containing more than one entries ends with a comma, like what you are doing for `deps`.

---

From https://docs.bazel.build/versions/master/skylark/build-style.html

> We let the **machine** take care of most formatting issues.

[`Buildifier`](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md), like `clang-format`, 
is just a command-line tool. Its README.md has detailed instructions for how to use it.

Don't know what blocks you.

Reply:
Yes, I just tried once following the readme and it didn't work.

Reply:
now fix it by hand